packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -186,6 +186,14 @@ contract DeployOwnership is Deploy {
         _callViaSafe({ _safe: safe, _target: address(safe), _data: abi.encodeCall(GuardManager.setGuard, (guard)) });
         console.log("LivenessGuard setup on SecurityCouncilSafe");
 
+        // Deploy and add the Liveness Module.
+        address livenessModule = deployLivenessModule();
+        _callViaSafe({
+            _safe: safe,
+            _target: address(safe),
+            _data: abi.encodeCall(ModuleManager.enableModule, (livenessModule))
+        });
```

Comment:
See that we moved the check into `DeployOwnership.t.sol`, but doesn't look like these tests are guaranteed to be ran prior to the deploy script's invocation - just in CI?

i.e., if you were to break these tests locally prior to invoking the deploy script, there would be no check that prevents the deploy script from executing, unless I'm missing some implicit check here.

Reply:
How would you suggest going about ensuring that tests run on the script anytime it is run?

Reply:
I think @clabby is suggesting copying the `require(_safe.getThreshold() >= getRequiredThreshold(owners.length))` check into the end of the script, instead of moving it only into a test

Reply:
I see. Do you want to see that as well? We previous agree on testing the script in this test file, I'm fine with adding a check here, but I'm concerned that the next comment will be that all of the checks in the test should be replicated here.

Reply:
I actually expected that the check would be moved to the script and was surprised it was only in the test, so I would suggest moving it. The argument against "the next comment will be that all of the checks in the test should be replicated here" is that this check is checking configuration of an instance, whereas other tests are checking intrinsic behavior of the contract. So I think just moving this one check into the script is sufficient

Reply:
ok, sgtm!